### PR TITLE
feat(site): make / the product landing page (#1021)

### LINKED ISSUE #42: Native Windows support
## PRD: Native Windows Support

**Problem**: dot-agent-deck only runs on macOS and Linux. Users on Windows must use WSL, which adds friction and limits discoverability. The codebase has three Unix-specific dependencies: Unix domain sockets for daemon IPC, `/dev/tty` for OSC 52 clipboard writes, and Unix-specific PTY signal handling.

**Solution**: Replace Unix domain sockets with cross-platform IPC (named pipes on Windows, Unix sockets elsewhere), add `conout$` fallback for clipboard, and handle platform-specific PTY differences. Ship Windows binaries (amd64, arm64) in releases.

**Detailed PRD**: See [prds/42-native-windows-support.md](https://github.com/vfarcic/dot-agent-deck/blob/main/prds/42-native-windows-support.md)

**Priority**: Medium

### LINKED ISSUE #1021: The landing page is a Docusaurus docs home, not a product website: design and build a real one — style exploration first
## Intent

`agent-deck.devopstoolkit.ai` should open on something that reads as a **product website** — a landing page that sells what Agent Deck is and who it is for. Today it opens on a Docusaurus home page: the docs' own front door, wearing the docs' chrome and the docs' visual language.

This issue covers designing and building that landing page. **The first task is to settle the style by looking at a few real candidates, not to start building one** — see Task 1.

## Where it stands today, measured on `main` at `98683b4b`

`site/src/pages/index.js` is 240 lines and renders, in order: a hero, a screenshot, a "Why Agent Deck" prose block, four feature cards, four "Design Principles" cards, and a Quick Start with per-OS tabs.

The content is not the problem — the "Why Agent Deck" section in particular is genuinely good positioning. **The presentation is.** The page is built entirely out of Docusaurus's own theme primitives:

- it renders inside `@theme/Layout`, so it inherits the docs navbar and the docs footer;
- every visual decision is an Infima utility class — `hero`, `hero__title`, `hero__subtitle`, `button button--secondary button--lg`, `container`, `row`, `col col--3`, `text--center`, `padding-horiz--md`;
- the only bespoke styling is `site/src/pages/index.module.css`, at 1460 bytes.

So it looks the way every Docusaurus site looks, because structurally it *is* one. That is the thing to change.

## Task 1 — settle the style first, by looking at real examples

**Do not start building the final page. Produce a handful of candidate directions, show them, and stop for a pick.**

What a candidate should be:

- **A real page in a browser, not a description.** Each direction gets its own route (`/style/a`, `/style/b`, …) so they can be opened side by side and judged the way a visitor would. A written description of a visual direction cannot be evaluated.
- **Built with this product's actual content** — the real tagline, the real positioning copy, a real screenshot. A direction that only looks good with placeholder text is not a direction, and the current page's copy is strong enough to carry a layout.
- **Attributed.** For each one, name the two or three sites it is drawing from and say in one line what it optimises for — e.g. stark developer-tool minimalism, dense technical credibility, or bold product marketing. That is what makes the choice discussable rather than a matter of taste.
- **Three or four of them, genuinely different.** Four variations on the same idea is one candidate presented four times.

**At least one candidate should abandon the Docusaurus theme entirely** — no `@theme/Layout`, its own header and footer — because that is the decision hiding underneath this whole issue and it should be made by looking rather than assumed either way. The docs are already mounted at `/docs` (`routeBasePath: 'docs'` in `site/docusaurus.config.js`), so `/` is free to diverge as far as we want. The open question is how far, and what the navigation between a marketing-styled `/` and a Docusaurus-styled `/docs` should feel like.

Then present them and **stop**. Nothing else in this issue starts until a direction is picked. When one is, delete the losing routes rather than leaving them in the tree.

## The rest of the work, once a direction is chosen

- [ ] Build the landing page in the chosen direction.
- [ ] Decide and implement the `/` ↔ `/docs` relationship: shared chrome, or a deliberate handoff.
- [ ] Make it work at phone width, and in **both** light and dark — the site has a theme toggle, so a single-background design fails half the visitors.
- [ ] Refresh the screenshots. `site/static/img/` carries seven PNGs and three JPGs, all but one dated May, so they predate a good deal of the current UI.
- [ ] Fix the content that is now false (below).
- [ ] Publish and look at it live, via the `publish-docs` project-local skill.

## Content on the current page that is already wrong

Worth fixing in the same pass, since this is the page being rebuilt:

- **The desktop app is missing entirely.** The tagline is "A terminal dashboard for running multiple AI coding agents in parallel", and nothing on the page mentions the desktop GUI — which now ships signed and notarized artifacts for macOS, Windows and Linux on every release (#740, #757). A visitor cannot learn from this page that it exists.
- **The supported-agent list is stale.** The page says "Works with Claude Code and OpenCode out of the box" and "Claude Code or OpenCode" under Design Principles. This repo's own `.dot-agent-deck.toml` runs four harnesses across two providers today — claude, pi, opencode and codex — and there are adapter PRDs open for Gemini (#211) and Aider (#212).
- **The Windows tab links to a closed issue.** It reads "Native Windows is coming soon" and links #42, which closed on 2026-07-15, as did #163. The open Windows work is #164 (release artifacts, e2e validation and docs). As written the page understates the state of Windows support and points at the wrong thread.
- **The Linux install instructions hedge**: "Install via Homebrew (if available)". Either it is the supported path on Linux or it is not; the page should say which.

## Constraints

- **Publishing runs through the existing path** — `docs-publish.yml` builds the site into a GHCR image tagged `main-<sha>` and bumps `site/helm/values.yaml` so Argo CD rolls it forward. The `publish-docs` skill is the supported way to ship this between releases. Nothing here should need a new deployment mechanism.
- **CLAUDE.md rule 11 applies**: this is user-facing, so it belongs under the published tree. `docs/develop/` is excluded from the build and must stay that way.
- **Rule 10**: no hard-wrapped Markdown prose in anything this touches.
- The `site/` build is pnpm-based and has no Rust in it, so the usual cargo gates are not the relevant ones here — but `cargo test-fast` and the clippy gate still apply to any commit, per rule 2.

## Related

- **#746 — design a project logo.** A real dependency rather than a cross-reference: a landing page whose job is to look like a proper product will be actively undermined by a placeholder mark, and that issue records that the two existing marks share no colour, no shape language and no concept, plus that `site/docusaurus.config.js:7` points at an `img/favicon.ico` that does not exist. Worth deciding whether the logo lands first or the two run together.
- **#765** — document installing the desktop GUI once it leaves alpha. Same gap as the desktop omission above, on the docs side.
- **#164** — the open Windows work the Quick Start tab should be pointing at.

### PR DESCRIPTION
**Pull request by vfarcic**
## What this changes

`agent-deck.devopstoolkit.ai` opened on a Docusaurus docs home rather than a product website. Issue #1021 asked for a style exploration first — four candidate directions on `/style/a` … `/style/d`, built with real content and attributed to the sites they drew from — and then a real landing page. The maintainer picked **direction C** (bold product marketing, drawing on linear.app / raycast.com / warp.dev); the three losing routes are deleted, as the issue requires.

- **A landing page at `/`**, replacing 240 lines of Infima utility classes with its own visual layer. All copy lives in one module, `site/src/data/landing-content.js`.
- **The `/` ↔ `/docs` relationship is decided: shared chrome.** The page keeps `@theme/Layout`, so the navbar and footer stay the docs' own and nothing moves when a visitor clicks through; everything between them is the landing page's. The closing band ("The door into the docs") says the handoff out loud and hands the visitor six specific pages. It is the one idea carried over from a losing candidate.
- **Four false-content items fixed**, each verified against the repo rather than taken from the issue text:
  - the desktop app was missing from the page entirely;
  - the agent list said "Claude Code and OpenCode" when the product ships five (Claude Code, OpenCode, Pi, Codex, Devin — `src/agent_registry.rs`);
  - the Windows tab linked the closed #42 instead of the open #164;
  - the Linux install hedged with "Homebrew (if available)" although the generated formula carries a full `on_linux` block for both architectures.
- **The issue's own desktop claim was wrong, and the page does not repeat it.** #1021 says the desktop artifacts ship "signed and notarized". They are an **unsigned alpha with no Windows bundle** — `release.yml:523` says so, and #757 closed as "free options only… the macOS warning stays". The page says alpha, unsigned, no Windows bundle, and claims what did ship: build provenance for the binaries and the desktop packages.
- **A WCAG AA failure fixed, site-wide.** `--ifm-color-primary: #4ecdc4` was set identically in both themes and fed `--ifm-link-color`, putting every docs-page link at 1.93:1 in light mode. The fix is narrow: `--ifm-link-color` is overridden to `#0a7c73` (5.07:1) in the light block only, and the primary ramp is untouched, so the docs are not restyled.
- **Ten screenshots refreshed or removed.** All were dated May or earlier. Two published images leaked a maintainer's home path (`/Users/viktorfarcic/…`) and one of them also showed an unrelated project's internal orchestration config — that one is deleted, the other recaptured. A new `reattach.png` backs the "walk away and come back" claim, and one recapture showed the New Agent dialog had changed shape entirely since the old image.
- **Phone width and both themes were measured, not asserted:** `scrollWidth == clientWidth` at 360px and 320px in both themes.

Changelog fragment: `changelog.d/1021.feature.md`.

## Review follow-up

Greptile's one finding (P2, positional `storyShots[i]` against `workflow`) was valid and is fixed in `31d417e0`: each `workflow` entry in `site/src/data/landing-content.js` now carries a `shot` key naming its screenshot, and `site/src/pages/index.js` reads `step.shot`. The rendered output is unchanged; the reproduction and the before/after build diff are on the resolved thread.

## Out of scope by design

Three follow-ups were filed from this work. Each is a `release.yml` or shared-token change that does not belong in a landing-page PR:

- #1152 — the release workflow attests the binaries but not the two checksum manifests (verified with a live 404 from `gh attestation verify`).
- #1153 — the release notes tell macOS users how to bypass Gatekeeper before how to verify the download.
- #1154 — latent: white-on-primary buttons would fail at 1.93:1.

## CLAUDE.md rule 12 — daemon / protocol / orchestration / hooks

Does not apply. This branch touches no daemon, protocol, orchestration or hook code — it is a static site plus images. No `PROTOCOL_VERSION` bump, no `CONTRACT_BREAKS` entry and no `.breaking.md` fragment, so no cross-version manual test is owed and none was run.

## How it was verified

The gates below were run on the final Rust tree by the workers who produced this branch. The tip is now `31d417e0`, two commits past that tree: `8d712b43` (the changelog fragment) and `31d417e0` (the review fix, two site files and no Rust). I re-ran `cargo fmt --check` and the clippy command myself at `8d712b43`; I did **not** run `cargo test-fast` or the site build locally on either commit. For `31d417e0` the coder reported `cargo fmt --check`, the clippy command, `cargo test-fast` (3299 passed, 0 skipped) and `npm run build` all passing, and CI on `31d417e0` is green on every check, including `build`, `build-macos`, `build-windows`, `security`, `e2e-deterministic` and `docs-build`.

| gate | result |
| --- | --- |
| `cargo fmt --check` | pass (workers; re-run by me at `8d712b43`; coder at `31d417e0`) |
| `cargo clippy --workspace --all-targets --features e2e,e2e-live -- -D warnings` | pass (workers; re-run by me at `8d712b43`; coder at `31d417e0`) |
| `cargo test-fast` | 3299 passed, 0 skipped (workers, final Rust tree; same count reported by the coder at `31d417e0`) |
| `cd site && npm run build` | pass (workers; coder at `31d417e0`; CI `docs-build` at `31d417e0`); `onBrokenLinks: 'throw'`, so every internal link resolves |

**Covering tests:** none by design. No Rust changed on this branch — the `src/agent_pty.rs` and `tests/` content that appears in a diff against the pre-rebase tip came from `main`'s own `delegate_034` fix (`364dc860`), which the rebase picked up. No TUI behavior changed, so no L1/L2 tests are owed under rule 4, and no e2e test was added or changed. That is also why there is no demo-reel clip: the demo-reel adapter reports `skipped: no e2e tests changed on this branch`.

Closes #1021

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: site/src/pages/index.js [hunk 1]
```diff
@@ -1,240 +1,361 @@
...
+ * its own copy describes -- 01 is the form `Ctrl+n` opens, 04 is the
+ * before/after detach pair -- and `landing-content.js` records why the two
+ * images these displaced left the page rather than moving elsewhere.
+ */
+const storyShots = [
+  screenshots.newPane,
+  screenshots.card,
+  screenshots.parallel,
+  screenshots.reattach,
+];
```
[the remark this answers is not in the document]
Comment (vfarcic) [reply site/src/pages/index.js]: Valid, and fixed in 31d417e0.

The two arrays met only by index, so the pairing was a convention rather than something the code stated. This branch is its own evidence: bc6f0abe exists because rows had already been mispaired by hand once.

**What changed** (2 files, 17 insertions, 15 deletions): each `workflow` entry in `site/src/data/landing-content.js` now carries a `shot` key naming its `screenshots` entry (`newPane`, `card`, `parallel`, `reattach`), `site/src/pages/index.js` reads `step.shot.src` / `.alt` / `.caption`, and the `storyShots` array and its comment are gone. The `i` in `workflow.map` is still used, but only to alternate the row layout (`i % 2`), not to choose content.

**Measured by the coder who made the change, not asserted:**

- **Reproduced against the old code.** Building with the `workflow` array reversed mispaired every row — "Walk away" rendered the New Agent form and "Open a pane" rendered the before/after detach pair — with build exit 0 and no warning. With the new code the same reversal keeps each row with its own frame.
- **Rendered output is unchanged.** A diff of the whole `site/build` tree between the two builds shows one differing line across all 17 HTML files, the content-hashed webpack runtime filename, which moves whenever any source byte changes. The four rendered story tuples (step, title, img src, alt, figcaption) are identical before and after.
- **No hydration risk.** The landing page's JS chunk moves by 18 bytes and its 372 string literals are an identical set between builds.
- **The failure mode is now loud in both directions.** A misspelled `shot` key is `undefined`, so the first property read throws a `TypeError` at build time. Before, a length mismatch was loud but a reorder was silent.

CI on 31d417e0 is green, including `docs-build` (the Docusaurus build with `onBrokenLinks: 'throw'`).
